wagtail_xliff_translation/tests/conftest.py

```diff
@@ -1,12 +1,104 @@
-# from .fixtures import *  # noqa
-from pytest_factoryboy import register
-from test_app.factories import PageFactory, PageWithRichTextFactory, PageWithRichTextFactory, LanguageFactory
-
-
-register(PageFactory)
-register(PageWithRichTextFactory)
-register(PageWithRichTextFactory)
-register(LanguageFactory)
```

Comment:
Don't we want to keep these ^? And drop `def page_factory():`, `def site_factory():`, `@pytest.fixture`, `def page_with_richtext_factory():` 

Because register is the documented way to create factory fixtures.

Then you can do in a test:

```
def some_test(page):
    # page is a page instance
```
OR
```
def some_test(page_factory):
    # page_factory is a factory you can call it with arguments to get a page instance
    page = page_factory(title="foo")
```